assets/js/vue/components/search/SearchForm.vue

```diff
@@ -117,6 +117,15 @@
                 submitted: false,
             }
         },
+        computed: {
+            displayTextBlock: function () {
+                if (typeof window.PELAGOS_TEMPLATE_PROPS !== 'undefined') {
+                    return window.PELAGOS_TEMPLATE_PROPS.hideSearchTextBlock;
+                } else {
+                    return false;
+                }
+            }
+        },
```

Comment:
Isn't this from another PR, it should show up right?

Reply:
Yes, that's weird

Reply:
Did you commit anything after?

Reply:
This may be some Github weirdness, because this is on `develop`